addons/html_editor/models/ir_qweb_fields.py

```diff
@@ -39,42 +39,53 @@ class IrQweb(models.AbstractModel):
     """
     _inherit = 'ir.qweb'
 
-    def _compile_node(self, el, compile_context, level):
-        snippet_key = compile_context.get('snippet-key')
+    def _compile_root(self, element, compile_context):
+        snippet_key = compile_context.pop('snippet-key', None)
+        sub_call_key = compile_context.pop('snippet-sub-call-key', None)
 
         template = compile_context['ref_name']
-        sub_call_key = compile_context.get('snippet-sub-call-key')
+
+        # Removes the attributes used by the "/website/snippet/filter_templates"
+        # controller and used only to be filtered without using irQweb rendering
+        for data_filter in [
+                'data-number-of-elements', 'data-number-of-elements-sm',
+                'data-number-of-elements-fetch', 'data-row-per-slide',
+                'data-arrow-position', 'data-extra-classes',
+                'data-extra-snippet-classes', 'data-container-classes',
+                'data-content-classes', 'data-column-classes', 'data-thumb',
+            ]:
+            element.attrib.pop(data_filter, None)
```

Comment:
Don't know if you finished adapting, but you didn't remove them (even though you did add them in website) :slightly_smiling_face: